lib/ansible/cli/doc.py

```diff
@@ -284,6 +282,13 @@ def _create_role_doc(self, role_names, roles_path, entry_point=None):
         return result
 
 
+def _doclink(url):
+    # assume that if it is relative, it is for docsite, ignore rest
+    if not url.startswith("http"):
```

Comment:
Hmm, this will break several existing relative links (that are/were optimized for the old and/or current HTML docsite generator).

How about:
```suggestion
    if not url.startswith("http") and not url.startswith(".."):
```
That should cover the most common cases.

Also there's another problem: **which** docsite? The ansible-core docsite, or the Ansible docsite?